src/third_party/har.js

```diff
@@ -105,11 +105,13 @@ exports.createHar = function (page, resources) {
             entry.time = endReply.time - request.time;
 
             if (entry.time > 3600000) {
-                // Found wierd bug (#150127455)
+                // Found weird bug (#150127455)
                 entry.debug = {};
                 entry.debug.endReplyTime = endReply.time.valueOf();
                 entry.debug.requestTime = request.time.valueOf();
-                entry.time -= 3600000;
+
+                request.time = new Date(request.time.getTime() + 3600000);
```

Comment:
request.time is "corrected" here, but we have used the original version already calculating entry.timings.wait. So that wait value must be also incorrect sometimes.

Reply:
We also specify entry.startedDateTime based on the original request.time, not the "corrected" version.